contributors.md

```diff
@@ -19,8 +19,12 @@ _(Do not modify this list unless you are a maintainer)_
 _(People who have participated in the challenges!)_
 
 - [RaidAndFade](https://github.com/raidandfade)
+
+- [sroehrl](https://github.com/sroehrl)
```

Comment:
Please remove the spacing around your link, The list will look nicer when it is condensed as we get more people.

Reply:
I suppose you are talking about the new lines? I changed it.